firebase_admin/__init__.py

```diff
@@ -257,7 +258,7 @@ def _lookup_project_id(self):
         if not project_id:
             try:
                 project_id = self._credential.project_id
-            except AttributeError:
+            except (AttributeError, DefaultCredentialsError):
                 pass
```

Comment:
Do we want to throw here or pass?

Reply:
We want to pass and look for other ways of getting a project_id.